importer/Uploader.py

```diff
@@ -152,9 +152,9 @@ def make_pywikibot_item(self, value, prop=None):
         if utils.string_is_q_item(value):
             val_item = self.make_q_item(value)
         elif prop == PROPS["image"]:
-            if not self.item_has_prop("image", self.wd_item):
-                if utils.file_is_on_commons(value):
-                    val_item = self.make_image_item(value)
+            if not self.item_has_prop("image", self.wd_item) and \
+                    utils.file_is_on_commons(value):
+                val_item = self.make_image_item(value)
         elif utils.tuple_is_coords(value) and prop == PROPS["coordinates"]:
             if not self.item_has_prop("coordinates", self.wd_item):
```

Comment:
Add a comment on this line that this is a temporary thing awaiting https://phabricator.wikimedia.org/T160282. After this you are good to go.